### ISSUE TITLE
Ubuntu 16.04 in OS Support Matrix, but no OBS repository exists

### ISSUE THREAD

**Issue by halmartin** [zeek/zeek#1242]
According to the [Operating System Support Matrix](https://github.com/zeek/zeek/wiki/Zeek-Operating-System-Support-Matrix) Ubuntu 16.04 is supported for Zeek LTS releases.

However, it appears that no repository exists for Ubuntu 16.04 in the OpenSuse Build service: https://download.opensuse.org/repositories/security:/zeek/

Which repository should one use if installing `zeek-lts` on 16.04?

**Comment by 0xxon** [comment]
Hi,

sorry - that was a mistake that I made when making the support matrix. Actually, for some reason that I could never figure out, the GCC that is used by OBS crashes when trying to build the LTS version of Zeek.

Since it has been a while since I tried - let me try again to see if something changed about that. I will report back in a few hours.

**Comment by 0xxon** [comment]
This sadly is still the case; on OBS, the compile of our LTS version fails with an internal compiler error:

```
[ 1096s] x86_64-linux-gnu-g++: internal compiler error: Segmentation fault (program cc1plus)
[ 1096s] Please submit a full bug report,
[ 1096s] with preprocessed source if appropriate.
[ 1096s] See <file:///usr/share/doc/gcc-5/README.Bugs> for instructions.
[ 1096s] aux/broker/bindings/python/CMakeFiles/_broker.dir/build.make:62: recipe for target 'aux/broker/bindings/python/CMakeFiles/_broker.dir/_broker.cpp.o' failed
[ 1096s] make[4]: *** [aux/broker/bindings/python/CMakeFiles/_broker.dir/_broker.cpp.o] Error 4
```

I will update the support matrix to reflect this. This sadly means that, for the moment, you will have to compile Zeek yourself to install it on Ubuntu 16.04.
